packages/@glimmer/benchmark-env/src/benchmark/create-registry.ts

```diff
@@ -51,7 +51,7 @@ export interface Registry {
     name: string,
     template: SerializedTemplateWithLazyBlock,
     component: T,
-    manager: ComponentManager<unknown, T>
+    manager: InternalComponentManager<unknown, T>
```

Comment:
Curious, why can't this use the "public" component manager?

Reply:
Sounds good, can you make an issue to track that)/

Reply:
Yup! Made it at: https://github.com/glimmerjs/glimmer-vm/issues/1201